test/test_cases/run_2d_moist_bubble.jl

```diff
@@ -125,7 +125,18 @@ function run_2d_moist_bubble(
         # mkpath(path)
         # cb_jld2 = JLD2Output(model, path, "moist_bubble", dt)
         # cb_set = CallbackSet(generate_callback(cb_jld2))
-        simulation = Simulation(model, stepper, dt = dt, tspan = (0.0, 500.0))#, callbacks = cb_set,)
+
+        path = ""
```

Comment:
maybe something like
```
        path = joinpath(@__DIR__, first(split(basename(@__FILE__), ".jl")))
        mkpath(path)
```
so that if people fail to specify a dir, it has a default.